src/pages/Collection/index.tsx

```
@@ -231,6 +616,72 @@ export const Collection: React.FC = () => {
+            break;
+          }
+          if (traitKeys.indexOf(search) !== -1) {
+            relevancy += 1;
+            break;
+          }
+          if (traitValues.indexOf(search) !== -1) {
+            relevancy += 1;
+            break;
+          }
+        }
+      } while (0);
```

Biome

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)